components/SnowflakeApp.js

```diff
@@ -89,10 +92,10 @@ const defaultState = (): SnowflakeAppState => {
       'CUSTOMER_VALUE': 0,
       'COMMUNITY': 0
     },
-    focusedTrackId: 'MOBILE'
```

Comment:
This line needs to be added back so in the default state, the Mobile section is selected.